php/router.php

```diff
@@ -36,6 +36,21 @@ public function postHandler($namespace, $endpoint, $callback)
             );
         }
 
+        public function codeHandler($namespace, $endpoint, $callback)
+        {
+            \register_rest_route(
+                $namespace,
+                $endpoint,
+                [
+                    'methods' => 'POST',
+                    'callback' => $callback,
+                    'permission_callback' => function () {
+                        return \current_user_can($this->capability);
+                    },
+                ]
+            );
+        }
+
```

Comment:
We shoulnd't need this, the `getHandler` should be enough

Reply:
Gotcha! Went ahead and removed the extra handler in [this commit](https://github.com/KevinBatdorf/code-block-pro/pull/252/commits/56ce39f60b3dbb67e9ba98bad3d88d83dbf61234)